raco/backends/sql/catalog.py

```diff
@@ -169,6 +172,9 @@ def _get_unary_sql(self, plan):
                 return sel
             return sel.group_by(*g)
 
+        elif isinstance(plan, algebra.Distinct):
```

Comment:
Yeah that's what I expected, I just wanted to be sure. Obviously Postgres can optimize away the subquery.